services/app-graphql/src/queries/fetchContractWithQuestions.graphql

```diff
@@ -0,0 +1,360 @@
+query fetchContractWithQuestions($input: FetchContractWithQuestionsInput!) {
```

Comment:
we will still need the fetchContractWIthQuestions.graphql, but if we implement the questions section of the ContractResolver we won't need a separate top level resolver for this, it will just be a call to fetchContract that includes Questions in the requested response